.editorconfig

```diff
@@ -0,0 +1,27 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,go,yml,json}]
+charset = utf-8
+
+[*.go]
+indent_style = space
+indent_size = 2
+
+[Makefile]
+indent_style = tab
+
+[*.js]
```

Comment:
We do not have JavaScript files in here, so please remove this to avoid confusion.